The circleCI failure points to a real problem (pasted below).  There is no  use in creating a new PR, but you can ignore the other failure.
```
/home/circleci/repo/venv/lib/python3.8/site-packages/numpy/__init__.py:docstring of numpy.cumsum:84: WARNING: Unexpected indentation.
/home/circleci/repo/venv/lib/python3.8/site-packages/numpy/__init__.py:docstring of numpy.cumsum:85: WARNING: Block quote ends without a blank line; unexpected unindent.
make: *** [Makefile:179: html-build] Error 1
```

--- a/numpy/core/fromnumeric.py
+++ b/numpy/core/fromnumeric.py
@@ -2523,13 +2523,10 @@ def cumsum(a, axis=None, dtype=None, out=None):
     ``cumsum(b)[-1]`` may not be equal to ``sum(b)``
 
     >>> b = np.array([1, 2e-9, 3e-9] * 1000000)
-    >>> np.cumsum(b)
-    array([1.00000000e+00, 1.00000000e+00, 1.00000000e+00, ...,
-           1.00000001e+06, 1.00000001e+06, 1.00000001e+06])
     >>> b.cumsum()[-1]
-    1000000.0050045159            # summation using cumsum() 
+    1000000.0050045159
     >>> b.sum()                    
-    1000000.0050000029            # summation using sum()
+    1000000.0050000029
 
     """
     return _wrapfunc(a, 'cumsum', axis=axis, dtype=dtype, out=out)

--- a/numpy/core/fromnumeric.py
+++ b/numpy/core/fromnumeric.py
@@ -2520,7 +2520,8 @@ def cumsum(a, axis=None, dtype=None, out=None):
     array([[ 1,  3,  6],
            [ 4,  9, 15]])
 
-    ``cumsum(b)[-1]`` may not be equal to ``sum(b)`` 
+    ``cumsum(b)[-1]`` may not be equal to ``sum(b)``
+
     >>> b = np.array([1, 2e-9, 3e-9] * 1000000)
     >>> np.cumsum(b)
     array([1.00000000e+00, 1.00000000e+00, 1.00000000e+00, ...,

